lib/org/chromium/webidl/BlinkIDLFile.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'BlinkIDLFile',
+  extends: 'org.chromium.webidl.GitilesIDLFile',
+
+  documentation: 'An IDL file accessible via Gitiles.',
```

Comment:
True. Turns out this file was a mistake. I was going to add some Blink-specific stuff but decided against it. Mistakenly got roped in with other files in the dir though.